lib/gooddata/models/project.rb

```diff
@@ -1178,16 +1178,9 @@ def objects_export(objs, options = {})
         return
       end
 
-      objs = objs.pmap { |obj| [obj, objects(obj)] }
-      if objs.any? { |_, obj| obj.nil? }
-        object = objs.select { |_, obj| obj.nil? }.map { |o, _| o }.join(', ')
-        error_message = "Exporting objects failed with messages. " \
-                        "Object #{object} could not be found."
-        fail ObjectsExportError, error_message
-      end
       export_payload = {
         :partialMDExport => {
-          :uris => objs.map { |_, obj| obj.uri },
+          :uris => objs,
```

Comment:
Does this mean we expect objs to be strings? In that case the documentation doesn't apply: https://github.com/gooddata/gooddata-ruby/pull/1152/files#diff-e097692ced3388707c4489714a5d7cd9R1168

What happens when objs is an array of mdobjects? Do we have a test for it? objects_export is a publicly documented method so we must be sure there is no regression.

Reply:
fixed